src/stream/mod.rs

```diff
@@ -69,6 +69,12 @@ enum StackItem {
     DictValue(Value),
 }
 
+/// Options for customizing serialization of XML plists.
+#[derive(Clone, Debug)]
+pub struct WriteOptions {
+    pub(crate) indent_str: &'static str,
```

Comment:
I don't think this needs to be `pub(crate)`. Private items should be accessible from child modules.